client/src/containers/HomepageContainer/index.js

```diff
@@ -115,11 +115,11 @@ class HomepageContainer extends React.Component {
                   titlePosition="bottom"
                   titleBackground="linear-gradient(to bottom, rgba(0,0,0,0.4) 0%,rgba(0,0,0,0.3) 70%,rgba(0,0,0,0) 100%)"
                 >
-                <img
-                  className="homepage-images"
-                  alt={index.toString()}
-                  src={fields.images[0].fields.file.url}
-                />
+                  <img
+                    className="homepage-images"
+                    alt={index.toString()}
+                    src={fields.images[0].fields.file.url}
```

Comment:
Careful accessing a property this deeply nested without checking for its existence first. It will throw an error in any of these properties do not exist.